docs/source/02-get-started/index.html.md.erb

```diff
@@ -11,21 +10,21 @@ This allows you to test that the product works on your local environment.
 ## Clone the repository onto your local machine
 
 To install the framework, clone the [sre-monitoring-as-code](https://github.com/HO-CTO/sre-monitoring-as-code) repository onto your local machine.
-This is currently a private repo. If you’d like access to the files, please email [Team email address here]().
-
-See [ReadMe](https://github.com/HO-CTO/sre-monitoring-as-code/tree/main/local) for Detailed Local Installation instructions.
+This is currently a public repo open to contributions.
 
 ## Install pre-requisites
 
 To start using Monitoring-as-Code, you also need to:
 
-- [install Prometheus](https://prometheus.io/download/)
-- [install Grafana](https://grafana.com/get/?plcmt=top-nav&cta=downloads)
-- [install Docker](https://www.docker.com/get-started/)
+- [Install Docker](https://www.docker.com/get-started/)
+- [Install Docker Compose](https://docs.docker.com/compose/install/)
 
 ## Test your installation
-You can use the monitoring-grafana-journey-view mixin file to test that the product works as expected. The mixin file uses the Monitoring-as-Code product itself as an input to test.
+You can use the mixin files within `monitoring-as-code/mixin-defs` to test MaC works as expected.  
+This is a more detailed explanation on creating an [example artifact here](https://github.com/HO-CTO/sre-monitoring-as-code/tree/main/monitoring-as-code).  
+Once you have created the artifact you can view them in a local Grafana and Prometheus environment using [these instructions](https://github.com/HO-CTO/sre-monitoring-as-code/tree/main/monitoring-as-code).  
```

Comment:
Rather than just linking to the readme files here, I think we should add code blocks for the 2 or 3 commands necessary to run and deploy MaC Locally, then link the the README for further info. Just saves someone new to the project from having to go searching, and allows a reader to run a few simple commands to get things up and running.

e.g.

```
cd monitoring-as-code/
docker build ....
./deploy.sh
```

```
cd local/
docker-compose up
```

Navigate to http://localhost:3000 to view the Grafana dashboards.